plugins/push/frontend/public/templates/push-notification-details.html

```diff
@@ -109,10 +107,10 @@ <h2> {{pushNotification.name }} </h2>
             <div class="cly-vue-push-notification-details-summary">
                 <div class="bu-columns">
                     <div class="bu-column bu-is-10">
-                        <cly-header title="Message Summary" class="cly-vue-push-notification-details-summary__header">
+                        <cly-header class="cly-vue-push-notification-details-summary__header">
                             <template v-slot:header-left>
                                 <h2> {{i18n('push-notification-details.summary-header')}} </h2>
-                                <cly-tooltip-icon :tooltip="i18n('push-notification.description')" icon="ion ion-help-circled"> </cly-tooltip-icon>
+                                <cly-tooltip-icon :tooltip="i18n('push-notification-details.summary-header-descriptionn')" icon="ion ion-help-circled" style="margin-left:8px"> </cly-tooltip-icon>
```

Comment:
Were double "n"s intentional? :slightly_smiling_face:

Reply:
No they werent. Thank you for catching it.